dataland-message-queue-utils/src/main/kotlin/org/dataland/datalandmessagequeueutils/cloudevents/CloudEventMessageHandler.kt

```diff
@@ -0,0 +1,73 @@
+package org.dataland.datalandmessagequeueutils.cloudevents
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.amqp.AmqpException
+import org.springframework.amqp.rabbit.core.RabbitTemplate
+import org.springframework.amqp.support.converter.MessagingMessageConverter
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.cloud.function.cloudevent.CloudEventMessageBuilder
+import org.springframework.cloud.function.cloudevent.CloudEventMessageUtils
+import org.springframework.messaging.MessageHeaders
+import org.springframework.stereotype.Component
+import org.springframework.util.MimeTypeUtils
+import org.springframework.amqp.core.Message as MessageMQ
+import org.springframework.amqp.core.MessageProperties as AMQPMessageProperties
+import org.springframework.messaging.Message as MessageResult
+
+/**
+ * Handling of messages in the CloudEvents format
+ * @param rabbitTemplate
+ * @param objectMapper
+ */
+
+@Component("CloudEventMessageHandler")
+class CloudEventMessageHandler(
+    private val rabbitTemplate: RabbitTemplate,
+    @Autowired var objectMapper: ObjectMapper,
```

Comment:
seemingly the objectMapper isn't needed here?!

Reply:
Deleted